source/quickinstallationguide/qig.rst

```diff
@@ -69,7 +69,7 @@ To complete this guide you'll need the following items:
 #. At least one computer which supports and has enabled hardware virtualization.
 
 #. An `CentOS 7.9 minimal x86_64 install ISO, on bootable media
-   <http://isoredirect.centos.org/centos/7/isos/x86_64/>`_
+   <https://vault.centos.org/7.9.2009/isos/x86_64/>`_
```

Comment:
URL update is fine. I think, docs should be updated to not include EL7 details further (as it'll be no longer supported from 4.20). cc @vishesh92